add-chain/testdata/superchain/configs/sepolia/expected_zorasep.yaml

```diff
@@ -0,0 +1,21 @@
+name: testchain_zorasep
+chain_id: 4206904
+public_rpc: http://awe.some.rpc
+sequencer_rpc: http://awe.some.seq.rpc
+explorer: https://awesomescan.org
+
+superchain_level: 1
+standard_chain_candidate: true # This is a temporary field which causes most of the standard validation checks to run on this chain
+superchain_time: null # Missing hardfork times are NOT yet inherited from superchain.yaml
+contracts_version_tag: op-contracts/v1.3.0 # Multi-Chain Prep (MCP) https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.3.0
```

Comment:
Diff from actual zorasep: I removed `""` from around `"op-contracts/v1.3.0"`

Reply:
Is this a change we want to make elsewhere too? What problem did it cause?

Reply:
It caused the comparison on expected vs actual config .yaml file to fail [here](https://github.com/ethereum-optimism/superchain-registry/blob/main/add-chain/e2e_test.go#L164).

We currently don't treat strings consistently in the config .yaml file. We have a regex to put double quotes around contract addresses [here](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/superchain.go#L201), but all other strings are expected to not have quotes. I imagine the actual zora sepolia config .yaml file was created before the `superchain.EnhanceYaml` function existed, so that's why it has double quotes around that address field